src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -128,22 +128,17 @@ public static function disable()
         }
     }
 
-    /**
-     * @return string|null
-     */
-    public function findFile($class)
+    public function findFile(string $class = null)
```

Comment:
not sure about the change: the annotation looked fine, and no default should be set

Reply:
reverted annotation and removed default null.